packages/ceramic-core/src/doctype/account-link/account-link-doctype-handler.ts

```diff
@@ -0,0 +1,135 @@
+import CID from 'cids'
+import {
+    AnchorProof,
+    AnchorStatus,
+    DocState, DoctypeConstructor,
+    DoctypeHandler,
+    InitOpts,
+    SignatureStatus
+} from "../../doctype"
+import { Context } from "../../context"
+import { validateLink } from "3id-blockchain-utils"
+import { AccountLinkDoctype, AccountLinkParams } from "./account-link-doctype"
+
+const DOCTYPE = 'account-link'
+
+export default class AccountLinkDoctypeHandler implements DoctypeHandler<AccountLinkDoctype> {
+    /**
+     * Gets doctype name
+     */
+    get name(): string {
+        return DOCTYPE
+    }
+
+    /**
+     * Gets doctype class
+     */
+    doctypeClass(): DoctypeConstructor {
+        return AccountLinkDoctype
+    }
+
+    /**
+     * Creates AccountLink instance
+     * @param params - Create parameters
+     * @param context - Ceramic context
+     * @param opts - Initialization options
+     */
+    async create(params: AccountLinkParams, context: Context, opts?: InitOpts): Promise<AccountLinkDoctype> {
+        return AccountLinkDoctype.create(params, context, opts);
+    }
+
+    /**
+     * Applies record (genesis|signed|anchor)
+     * @param record - Record to be applied
+     * @param cid - Record CID
+     * @param context - Ceramic context
+     * @param state - Document state
+     */
+    async applyRecord(record: any, cid: CID, context: Context, state?: DocState): Promise<DocState> {
```

Comment:
I know we changed it to only have `applyRecord` instead of having three separate methods. Do you think that still makes sense or is three separate methods more clear?